app/components/pages/createPoll/index.js

```diff
@@ -0,0 +1,234 @@
+                  </Box>
+
+                  <Box
+                    display="flex"
+                    justifyContent="space-between"
+                    flexDirection={['column', 'row']}
+                  >
+                    <Box width={[1, 1 / 2]} mt={10} px={3}>
+                      <Input
+                        OutlinedInputPlaceholder={inputLabelName}
+                        name="textFieldName"
+                        variant="outlined"
+                        Icon={PersonIcon}
+                        appendIcon
+                      />
+                    </Box>
+                    <Box flex="1" mt={10} px={3}>
+                      <CustomDatePicker
+                        name="startDate"
+                        label="Start Date"
+                        className={classes.CustomFullWidth}
+                      />
+                    </Box>
+                  </Box>
+
+                  <Box
+                    display="flex"
+                    justifyContent="space-between"
+                    flexDirection={['column', 'row']}
+                  >
+                    <Box flex="1" mt={10} px={3}>
+                      <CustomDatePicker
+                        label="End Date"
+                        name="endDate"
+                        className={classes.CustomFullWidth}
+                      />
+                    </Box>
+                    <Box width={[1, 1 / 2]} mt={10} px={3}>
+                      <Input
+                        id="standard-basic"
+                        OutlinedInputPlaceholder={inputLabelQuestion}
+                        name="textFieldQuestion"
+                        variant="outlined"
+                        Icon={HelpOutlineIcon}
+                        appendIcon
+                      />
+                    </Box>
+                  </Box>
+                  <FieldArray
+                    name="options"
+                    render={(arrayHelpers) => (
+                      <Box>
+                        <Box
+                          display="flex"
+                          justifyContent="space-between"
+                          flexWrap="wrap"
+                          flexDirection={['column', 'row']}
+                        >
+                          {values.options.map((option, index) => (
+                            <Box
+                              key={option.id}
+                              width={[1, 1 / 2]}
+                              mt={10}
+                              px={3}
+                              position="relative"
+                              display="flex"
+                            >
+                              <Box flex="1">
+                                <Input
+                                  OutlinedInputPlaceholder={`Option ${
+                                    index + 1
+                                  }`}
+                                  name={`options.${index}`}
```

Comment:
what's options.${index} doing it doesn't makes sense is options object here?

Reply:
its array which contains the number of options created onclick method